### DISCUSSION TITLE
Noco shows only dbo tables [MSSQL]

### DISCUSSION THREAD

**Discussion by MiroslavMikus** [nocodb/nocodb#906] [Discord Threads]
Node: **v12.22.6**
Arch: **x64**
Platform: **linux**
Docker: **true**
Database: **mssql**
> `select @@Version`: Microsoft SQL Server 2019 (RTM-CU14) (KB5007182) - 15.0.4188.2 (X64)  	Nov  3 2021 19:19:51  	Copyright (C) 2019 Microsoft 
Corporation 	Express Edition (64-bit) on Linux (Ubuntu 20.04.3 LTS) <X64>

ProjectOnRootDB: **false**
RootDB: **sqlite3**
PackageVersion: **0.83.6**

**Steps To Reproduce**
1. Connect to MSSQL with multiple schemas

**Expected behavior**
I would like to see all my tables 

**Screenshots**
![image](https://user-images.githubusercontent.com/29312172/147849165-c8e4c98a-d23a-4cc6-940a-245c96c81237.png)

Thanks up front
Best Miro

**Comment by o1lab** [comment]
Connecting to non dbo schema can be done like below. Please try it out.

https://www.youtube.com/watch?v=rS0NRwRCjx0

**Comment by RachelAmbler** [comment]
This is not working for me.  Please see bug #1342
